src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -0,0 +1,90 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline asset` commands:
+    * snapshot
+    * upload
+    * diff
+    * download
+"""
+
+import click
+
+from .._common import _handle_error
+
+
+@click.group(name="asset")
+@_handle_error
+def cli_asset():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_asset.command(name="snapshot")
+@click.option("--root-dir", help="The root directory to snapshot. ")
+@click.option("--manifest-out-dir", help="Path to directory for manifest output. ")
```

Comment:
How is this different from other CLIs' manifest option? They all seem to be path to manifest folder according to help. Let's keep them consistent if they're for the same thing.

Reply:
Made this more clear; it's supposed to be the destination where the created manifest will end up at, in case user wants to specify it.